optimize_SCTL_si.m

```diff
@@ -55,7 +55,7 @@
     try
     [bhat,OBJ,~,~,~,~,~] = fmincon(...
         @(b) objective_SCTL_s1(model,data,beta,b,delta,s,i,options,1),...
-        bhat_0,[],[],[],[],-10*ones(length(bhat_0),1),10*ones(length(bhat_0),1),[],options_fmincon);
+        bhat_0,[],[],[],[],-15*ones(length(bhat_0),1),10*ones(length(bhat_0),1),[],options_fmincon);
```

Comment:
shall we keep them symmetric?
```suggestion
        bhat_0,[],[],[],[],-15*ones(length(bhat_0),1),15*ones(length(bhat_0),1),[],options_fmincon);
```

Reply:
ah... sure, this just because once the optima hits the lower boundary. Maybe we set both to 15?

Reply:
yes!